nilearn/mass_univariate/tests/test__utils.py

```diff
@@ -0,0 +1,59 @@
+import math
+
+import numpy as np
+
+from nilearn.mass_univariate._utils import _null_to_p
+
+
+def test_null_to_p_float():
+    """Test _null_to_p with single float input."""
+    null = [-10, -9, -9, -3, -2, -1, -1, 0, 1, 1, 1, 2, 3, 3, 4, 4, 7, 8, 8, 9]
+
+    # Left/lower-tailed
+    assert math.isclose(_null_to_p(9, null, alternative='smaller'), 0.95)
+    assert math.isclose(_null_to_p(-9, null, alternative='smaller'), 0.15)
```

Comment:
```suggestion
    assert math.isclose(_null_to_p(-9, null, alternative='smaller'), 0.05)
```

Reply:
This won't pass though. There are 3/20 (0.15) elements in the null array that are less than or equal to -9.

Reply:
Sorry, I misunderstood the test.